client/js/views/Lobby.vue

```diff
@@ -57,7 +66,7 @@ export default {
     };
   },
   computed: {
-    ...mapGetters(['opponentIsReady', 'opponentUsername']),
+    ...mapGetters(['opponentIsReady', 'opponentUsername', 'isRanked']),
```

Comment:
No, and maybe this is me being naive but I always tend to gravitate to getters for the caching. That said we’re not actually mutating the value or calculating anything so I can switch it back since it’s not really doing anything here.